Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest:5000, host:5000
+ config.vm.provision "shell", privileged: false, inline: <<-SHELL
+ sudo apt-get update
+ # TODO: Install pyenv prerequisites
+  sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+  libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+  xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+ # TODO: Install pyenv
+   git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+   echo 'export PYENV_ROOT="$HOME/.pyenv"' >> ~/.profile
+   echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> ~/.profile
+   echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.profile
+   curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+   exec "$SHELL"  
+ SHELL
+  config.trigger.after :up do |trigger|
+    trigger.name = "Launching App"
+    trigger.info = "Running the TODO app setup script"
+    trigger.run_remote = {privileged: false, inline: "
+    # Install dependencies and launch
+      pyenv install 3.8.5
+      pyenv global 3.8.5
+      echo 'pyenv set'
+      cd /
+      cd /vagrant
```

Comment:
`cd /` does nothing when you immediately cd to a different absolute path on the next line.

`cd /vagrant` means go to the vagrant folder in the root of the file system. Did you mistake it for `cd vagrant` which would mean go into the vagrant folder from the current directory?